### PR TITLE
IGVF-1422 Schema property search

### PR DESCRIPTION
**Pull request by forresttanaka**

### CODE REVIEW

File: components/profiles.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  setSearchTerm,
+  searchMode,
+  className = "",
+}) {
+  return (
+    <div className={`relative grow ${className}`}>
+      <TextField
+        name="schema-search"
+        value={searchTerm}
+        type="search"
```
[the remark this answers is not in the document]
Comment (forresttanaka) [reply components/profiles.js:22]: I just noticed that in Safari, but not in Chrome or Firefox. Not sure if you can turn that off in Safari. Did you see this in Chrome?

**Review by mingjiecn — CHANGES_REQUESTED** [review]
Please check the comment above. I think the XCircleIcon should be removed, or any update to just keep one X icon.

File: components/profiles.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (mingjiecn) [reply components/profiles.js:22]: I saw it in Chrome too.

**Comment by forresttanaka** [comment]
@mingjiecn OK, this issue leaves devs with no _great_ choices. You can either 1) remove the custom icon and rely on the automatic browser-supplied icons or 2) hide the automatic browser-supplied icons and use a custom icon.

The problem with 1) is that Firefox never implemented this, so people don’t have _any_ clear icon there. The problem with 2) is that these prefixed CSS attributes to hide the automatic icon could change in the future, breaking this work around.

This prefix has been around for years with no change though. So we have the choice of immediately breaking Firefox by relying on the automatic icons, or possibly breaking the appearance in future Chrome and Safari if this workaround gets broken. I felt like potentially breaking the future makes the better option.

I linked to one of the discussions I found about this in the globals.css file.

**Review by mingjiecn — APPROVED** [review]

**Pull request merged**